shared/src/test/diff/qq/NuSyntax.mls

```diff
@@ -55,16 +55,36 @@ run(x `=> x)(42)
 //│ Parsed: code"+(${code"+(${code"1"}, ${code"*(${code"2"}, ${code"3"},)"},)"}, ${code"4"},)";
 //│ Code[Int, nothing]
 
+:e
+:p
+1 + `2 `+ `3
+//│ |1| |+| |`|2| |`|+| |`|3|
+//│ AST: TypingUnit(Quoted(App(Var(+), Tup(_: Unquoted(App(Var(+), Tup(_: IntLit(1), _: Quoted(IntLit(2))))), _: Unquoted(Quoted(IntLit(3)))))))
+//│ Parsed: code"+(${+(1, code"2",)}, ${code"3"},)";
+//│ ╔══[ERROR] Type mismatch in operator application:
+//│ ║  l.60: 	1 + `2 `+ `3
+//│ ║        	^^^^^^
+//│ ╟── quasiquote of type `Code[2, nothing]` is not an instance of `Int`
+//│ ║  l.60: 	1 + `2 `+ `3
+//│ ╙──      	     ^
+//│ ╔══[ERROR] Type mismatch in unquote:
+//│ ║  l.60: 	1 + `2 `+ `3
+//│ ║        	^^^^^^
+//│ ╙── operator application of type `Int` is not an instance of `Code[?a, ?b]`
+//│ Code[Int, nothing]
+
 :e
 :p
 `1 `+ `2 + 3
 //│ |`|1| |`|+| |`|2| |+| |3|
 //│ AST: TypingUnit(App(Var(+), Tup(_: Quoted(App(Var(+), Tup(_: Unquoted(Quoted(IntLit(1))), _: Unquoted(Quoted(IntLit(2)))))), _: IntLit(3))))
 //│ Parsed: +(code"+(${code"1"}, ${code"2"},)", 3,);
-//│ ╔══[ERROR] Type mismatch in application:
-//│ ║  l.60: 	`1 `+ `2 + 3
+//│ ╔══[ERROR] Type mismatch in operator application:
+//│ ║  l.78: 	`1 `+ `2 + 3
 //│ ║        	^^^^^^^^^^^^
-//│ ╙── expression of type `Code[?a, ?b | ?c]` is not an instance of `Int`
+//│ ╟── quasiquote of type `Code[?a, ?b | ?c]` is not an instance of `Int`
```

Comment:
Instead of calling this "quasiquote" you should just call it "code fragment"

Reply:
I've Updated it.